jscomp/core/jsoo_main.ml

```diff
@@ -75,6 +75,14 @@ let implementation ~use_super_errors prefix impl str  : Js.Unsafe.obj =
   let env = Compmisc.initial_env() in (* Question ?? *)
   let finalenv = ref Env.empty in
   let types_signature = ref [] in
+  if use_super_errors then begin
+    Misc.Color.setup Clflags.Always;
+    Super_main.setup ();
+  end;
+
+  (* copied over from Bsb_warning.default_warning_flag *)
+  Warnings.parse_options false "-30-40+6+7+27+32..39+44+45+101";
+
```

Comment:
lgtm, shall we share the magic string to make it easy to sync up?

Reply:
Done